pyvista/core/filters/data_set.py

```diff
@@ -2455,6 +2456,13 @@ def cell_data_to_point_data(self, pass_cell_data=False, progress_bar=False):
         >>> surf.plot(scalars='Area')
 
         """
+        if pass_cell_data is False and 'pass_cell_arrays' in kwargs:
+            pass_cell_data = kwargs.pop('pass_cell_arrays')
+            warnings.warn(
+                'Use of `pass_cell_arrays` is deprecated. Use `pass_cell_data` instead.',
+                PyVistaFutureWarning
```

Comment:
Arguably if this is a deprecation, it should be a `DeprecationWarning`.

Reply:
Although [`DeprecationWarning`s are almost always suppressed](https://docs.python.org/3/library/warnings.html#default-warning-filter). Do we have pyvista-specific deprecation warnings?

Reply:
Ah yes, It should use `PyVistaDeprecationWarning` as I mentioned in https://github.com/pyvista/pyvista/pull/3412#pullrequestreview-1131535378